docs/README.md

```diff
@@ -61,9 +67,12 @@ solve them.
 
```

Comment:
Instead of this line:
- [NYI](./README.md)

What about listing our current feature flags and their descriptions?

Reply:
Added a feature flag reference

Reply:
@austinlparker I think we can document the hero scenario now too?

Reply:
I think the feature flag docs need to be updated too. Maybe we should add this in subsequent PR to avoid overload here?

Reply:
Created an issue to track this separately